cfme/infrastructure/config_management.py

```diff
@@ -520,3 +431,103 @@ def __init__(self, name=None, url=None, ssl=None, credentials=None, key=None):
         self.ssl = ssl
         self.credentials = credentials
         self.key = key or name
+
+
+@navigator.register(ConfigManager, 'All')
+class MgrAll(CFMENavigateStep):
```

Comment:
It doesn't matter what's the name of those objects. 
But PyCharm complains when finds two classes with the same name in one file.
I suppose there may be some issues with interpreter as well if I leave both classes with All name.

Do you think it makes sense to rename all classes ?